<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="login_Button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>go in!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLine">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>pasword</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginLine">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLabel" name="auth_text">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>input login password</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
